src/maskAnalysis/__tests__/getFeret.test.ts

```diff
@@ -47,6 +70,28 @@ test('mask with only 1 pixel', () => {
           { column: 2, row: 1 },
         ],
       },
+      lines: {
+        minDiameter: [
+          [
+            { column: 1, row: 0 },
+            { column: 2, row: 0 },
+          ],
+          [
+            { column: 1, row: 1 },
+            { column: 2, row: 1 },
+          ],
+        ],
+        maxDiameter: [
+          [
+            { column: 1.5000000000000002, row: -0.5 },
```

Comment:
The principle of using `toBedeepCloseTo` is to not have to specify all the decimals in tests